### ISSUE TITLE
Chalet fails to execute git

### ISSUE THREAD

**Issue by thatbakamono** [chalet-org/chalet#157]
Chalet fails to execute git because the path located by chalet is missing a slash.
```
ERRORS: There was a problem fetching one or more dependencies.
   Error fetching git dependency: fmt
   Subprocess: Executable not found: C:/Program Files/Git/bingit.exe
```

**Comment by thatbakamono** [comment]
Nope, the slash isn't present.
`"git": "C:/Program Files/Git/bingit.exe",`

**Comment by rewrking** [comment]
That's odd. I'm not able to reproduce that on my windows machine, but I'll do some more digging. The workaround for now is to just add the slash in `.chaletrc`.

**Comment by thatbakamono** [comment]
I think I know what is the issue here. I played with my PATH a bit and actually managed to "fix" the problem. With `C:\Program Files\Git\cmd` present in my PATH (it's present by default as it's required for `git` to work in the command line) chalet sets `git` to `C:/Program Files/Git/bingit.exe` but if I remove `C:\Program Files\Git\cmd` and add `C:\Program Files\Git` instead, it actually sets `git` to the correct path - `C:/Program Files/Git/bin/git.exe`. If I were to guess... I would say it's a off by one error in substring but that's just a guess.

**Comment by rewrking** [comment]
Aaaaah yes, that's exactly what's happening. It's in `src/SettingsJson/SettingsJsonParser.cpp:292`. Chalet tries really hard to find git on Windows, and instead of using the explicit "cmd" or "mingw64/bin" version of git, it switches to the base one. I'll have this fixed.

**Comment by rewrking** [comment]
Fixed in [v0.4.1](https://github.com/chalet-org/chalet/releases/tag/v0.4.1)
